index.html

```diff
@@ -214,17 +201,29 @@ <h2>Languajes
     </div>
     <div class="frameworks">
         Frameworks
+        <button class="right1">
         <i class="fa-solid fa-angle-right"></i>
-    </div>
-    <div>
-        <hr>
+        </button>
     </div>
     <div class="skills">
         Skills
+        <button class="right2">
         <i class="fa-solid fa-angle-right"></i>
-        <div>
-            <hr>
-        </div>
+        </button>
+    </div>
+</section>
+
+<section class="form-section">
+    <div class="form">
+        <h2 class="form-title-2">Contact me</h2>
+        <p>If you have an application you are interested in developing, a feature 
+            that you need built or a project that needs coding. I’d love to help with it </p>
+        <form class="form-media">
```

Comment:
- [ **OPTIONAL BUT RECOMMENDED** ] I think it would be better to add the [Formspree service](https://formspree.io/html) that will help you to collect data sent from your form. You can start by creating a [Formspree](https://formspree.io/register) account, then create a `new form` of your project and use its ID in the `<form>` tag which will look like this:
```bash
<form action="https://formspree.io/f/{form_id}" method="post">
   code of the form
</form>
```